contracts/LiquidationsPeripheral.vy

```diff
@@ -896,3 +896,48 @@ def adminWithdrawal(_walletAddress: address, _collateralAddress: address, _token
         _tokenId,
         _walletAddress
     )
+
+@external
+def adminLiquidation(_principal: uint256, _interestAmount: uint256, _collateralAddress: address, _tokenId: uint256):
+    assert msg.sender == self.owner, "msg.sender is not the owner"
+    assert IERC721(_collateralAddress).ownerOf(_tokenId) != ICollateralVaultPeripheral(self.collateralVaultPeripheralAddress).collateralVaultCoreAddress(), "collateral still owned by vault"
+
+    liquidation: Liquidation = ILiquidationsCore(self.liquidationsCoreAddress).getLiquidation(_collateralAddress, _tokenId)
+    assert liquidation.lid != empty(bytes32), 'collateral not in liquidation'
+    assert block.timestamp > liquidation.gracePeriodMaturity, "liquidation in grace period"
```

Comment:
We can remove this assertion since we only care about the `lenderPeriodMaturity` here, it is always higher than the `gracePeriodMaturity`
```suggestion
```

Reply:
Ok, also one test can be dropped 👍